setup.py

```diff
@@ -48,11 +48,6 @@ def run(self):
                         'models/default_TEM_model/*',
                         'data_test/*'],
     },
-    extras_require={
-        'docs': ['sphinx>=1.6',
-                 'sphinx_rtd_theme>=0.2.4',
-                 'recommonmark'],
-    },
```

Comment:
Does ADS have instructions somewhere for building the docs locally? (Installing Sphinx requirements, running `make html`, etc.)

If so, those instructions might need to be updated somewhere, since ADS is now using a requirements file rather than the `[docs]` extras.)

Reply:
I'm not sure, @mariehbourget do you know? I can re-implement this using feature using the current versions, if you'd like.

Reply:
Up to you! I just wanted to make sure nothing fell out of date if there were existing instructions hiding somewhere.

Reply:
The only instructions in the wiki related to documentation does not touch this, so it may have been never documented. I've never build the doc locally, but maybe others have been using it. I could check on our slack.